### PR TITLE
ztp: cleanup sourceCRs to be used for platform upgrade using PGT 

### PR DESCRIPTION
**Pull request by nishant-parekh**
- remove ClusterVersionUpgradeGraph.yml from sourceCRs
 - remove spec.desiredUpdate and status sections to be added in using PGT overlays
Signed-off-by: Nishant Parekh <nparekh@redhat.com>

### CODE REVIEW

File: ztp/source-crs/ClusterVersion.yaml [hunk 1]
```diff
@@ -7,10 +7,13 @@ metadata:
...
   upstream: https://api.openshift.com/api/upgrades_info/v1/graph # default upstream.
-  desiredUpdate:
-    force: false
-    version: $version
-status:
-  history:
-    -  version: $version
-       state: "Completed"
+
+# add these changes using PGT overlay to generate the necessary policy to verify platform upgrade.
```
Comment (Missxiaoguo) [inline ztp/source-crs/ClusterVersion.yaml:11]: Looks good to me except a small update
```suggestion
# add these changes using PGT overlay to generate the policy to perform and verify platform upgrade.
```

Comment (nishant-parekh) [reply ztp/source-crs/ClusterVersion.yaml]: updated

**Review by serngawy — APPROVED** [review]

**Pull request merged**